code/modules/mob/language/uplifts.dm

```diff
@@ -24,24 +24,26 @@
 
 /datum/language/corvid
 	name = LANGUAGE_CORVID
-	desc = "A trilling language spoken by Neo-Corvidae."
+	desc = "A rough, loud language spoken by Neo-Corvidae."
 	speech_verb = "chirps"
-	ask_verb = "chirrups"
-	exclaim_verb = "trills"
+	ask_verb = "rattles"
+	exclaim_verb = "calls"
 	colour = "alien"
 	key = "v"
 	flags = WHITELISTED
 	space_chance = 50
 	syllables = list(
-			"ca", "ra", "ma", "sa", "na", "ta", "la", "sha", "scha", "a", "a",
-			"ce", "re", "me", "se", "ne", "te", "le", "she", "sche", "e", "e",
-			"ci", "ri", "mi", "si", "ni", "ti", "li", "shi", "schi", "i", "i"
+			"ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah",
+			"skre","ahk","ek","rawk","kraa","ii","kri","ka"
```

Comment:
Oh god is this vox pidgin's syllable list?

Reply:
yes, I thought it'd be better than whatever was there before (Which sounded more like a bird whispering in your ear or something to that effect)

Reply:
Neo corvid vox confirmed.